File: c#-2/exibirDesconto.cs
using System;

class exibirDesconto 
{
  public static void Main (string[] args) 
  {
    string nome;
    double preco, desconto, total;

    Console.WriteLine("Digite o nome do seu produto");
    nome = Convert.ToString(Console.ReadLine());

    Console.WriteLine("Digite o preço do seu produto");
    preco = Convert.ToDouble(Console.ReadLine());

    Console.WriteLine("O desconto do produto é de");
    desconto = Convert.ToDouble(Console.ReadLine());

    if(desconto == 0) {
      Console.WriteLine("O " + nome + " o valor do produto foi " + preco);
    }
    else {
      total = preco - (preco * (desconto / 100));
      Console.WriteLine("O " + nome + " o valor total da compra foi de " + total);
    }
  }
}

File: js-lista2/exibirDesconto.js
let produto = prompt("Digite o nome do produto: ");
let preco = prompt("Digite seu o preço do produto: ");
let desconto = prompt("Digite a desconto: ");

if(desconto == 0) {
  console.log("O valor total do " + produto + " foi de " + preco);
} else { 
  let total = Number(preco) - (Number(preco) * (Number(desconto) / 100));
  console.log("O valor foi de: " + total);
}